data/json/items/comestibles/meat_dishes.json

```diff
@@ -588,7 +591,7 @@
     "volume": 1,
     "charges": 4,
     "fun": 1,
-    "vitamins": [ [ "vitA", 3 ], [ "iron", 4 ], [ "vitB", 22 ] ]
```

Comment:
why are these values the same?

Reply:
On the sheet canned chicken's values were for 216g, in-game its 54 which is a fourth, so I divided the values on the sheet by 4 to get these, which ended up being the same.